transmogrifier/sources/ead.py

```diff
@@ -184,15 +184,59 @@ def get_optional_fields(self, xml: Tag) -> Optional[dict]:
                     note_value.append(subelement_value)
             if note_value:
                 note_head_element = note_element.find("head", string=True)
-                note = timdex.Note(
-                    value=note_value,
-                    kind=(
-                        note_head_element.string
-                        if note_head_element
-                        else self.crosswalk_type_value(note_element.name)
-                    ),
+                fields.setdefault("notes", []).append(
+                    timdex.Note(
+                        value=note_value,
+                        kind=(
+                            note_head_element.string
+                            if note_head_element
+                            else self.crosswalk_type_value(note_element.name)
+                        ),
+                    )
+                )
+
+        # numbering field not used in EAD
+
+        # physical_description
+        physical_descriptions = []
+        for physical_description_element in collection_description_did.find_all(
+            "physdesc", recursive=False
+        ):
+            if physical_description_value := self.create_string_from_mixed_value(
+                physical_description_element
+            ):
+                physical_descriptions.append(physical_description_value)
+        if physical_descriptions:
+            fields["physical_description"] = ". ".join(physical_descriptions)
+
+        # publication_frequency field not used in EAD
+
+        # publication_information
+        if publication_element := collection_description_did.find("repository"):
+            if publication_value := self.create_string_from_mixed_value(
+                publication_element, " "
+            ):
+                fields["publication_information"] = [publication_value]
+
+        # related_items
+        for related_item_element in collection_description.find_all(
+            ["altformavail", "otherfindaid", "relatedmaterial", "separatedmaterial"],
```

Comment:
`otherfindaid`: the majority of the data seems like it's intended for staff, so I'm not sure how useful it is to include this field.

Reply:
`relatedmaterial`: again for most of these the `<head>` value doesn't describe a relationship, so doesn't really fit that field value. I'm not seeing a consistent place to pull the `relationship` from in our data, so we can just exclude it (I don't think a relationship of `Related Materials` is useful since it doesn't add any new information).

In addition, for many records this field contains either several `<p>` tags or in some cases an actual list, and each of those individual paragraphs/list items represent a different related item. We should represent them that way in the TIMDEX record.

Reply:
`separatedmaterial`: for this field the crosswalked type value seems like a better fit for `relationship` than the `<head>` tag.

Reply:
1. Updated crosswalk value and created the spike ticket for URLs
2. Removed
3. Moved this to a separate parsing block
4. Done